src/Microsoft.DotNet.XHarness.Apple/AppRunnerBase.cs

```diff
@@ -76,26 +76,18 @@ protected async Task<string> FindDevice(TestTargetOs target)
                 entireFile: false,
                 LogType.SystemLog);
 
+            // We need to make the binary executable
             var binaryPath = Path.Combine(appInfo.AppPath, "Contents", "MacOS", appInfo.BundleExecutable ?? appInfo.AppName);
-            var arguments = new List<string>();
-
             if (File.Exists(binaryPath))
             {
-                // We need to make the binary executable
-                var result = await _processManager.ExecuteCommandAsync("chmod", new[] { "+x", binaryPath }, _mainLog, TimeSpan.FromSeconds(10), cancellationToken: cancellationToken);
-
-                if (!result.Succeeded)
-                {
-                    _mainLog.WriteLine($"Failed to make the binary at {binaryPath} executable, the run might fail");
-                }
+                await _processManager.ExecuteCommandAsync("chmod", new[] { "+x", binaryPath }, _mainLog, TimeSpan.FromSeconds(10), cancellationToken: cancellationToken);
```

Comment:
I'd do a 'a+x'

Reply:
I think the smallest possible change to the file system that will make it is better?